libbeat/outputs/elasticsearch/config_otel_test.go

```diff
@@ -20,69 +20,186 @@ package elasticsearch
 import (
 	_ "embed"
 	"fmt"
-	"os"
-	"path/filepath"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"go.opentelemetry.io/collector/confmap/confmaptest"
+	"go.opentelemetry.io/collector/confmap"
 	"gopkg.in/yaml.v2"
 
 	"github.com/elastic/elastic-agent-libs/config"
 )
 
 func TestToOtelConfig(t *testing.T) {
 
-	tests := []struct {
-		name      string
-		input     string
-		expOutput string
-		experr    bool
-	}{
-		{
-			name:      "basic elasticsearch input",
-			input:     "basic.yml",
-			expOutput: "basic-op.yml",
-			experr:    false,
-		},
-	}
-
-	for _, test := range tests {
-		t.Run(test.name, func(t *testing.T) {
-			rawConf, err := loadBeatConf(filepath.Join("testdata", test.input))
-			require.NoError(t, err)
-			beatCfg := config.MustNewConfigFrom(rawConf)
-
-			otelCfg, err := ToOTelConfig(beatCfg)
-			require.NoError(t, err, "could not convert beat config to otel ES config")
-
-			expectedValue, err := confmaptest.LoadConf(filepath.Join("testdata", test.expOutput))
-			require.NoError(t, err)
-			want, err := yaml.Marshal(expectedValue.ToStringMap())
-			require.NoError(t, err)
-
-			got, err := yaml.Marshal(otelCfg)
-			require.NoError(t, err)
-
-			assert.Equal(t, string(want), string(got))
-
-		})
-	}
+	t.Run("basic config translation", func(t *testing.T) {
+		basicInput := `
+hosts:
+  - localhost:9200
+  - localhost:9300
+protocol: http
+path: /foo/bar
+username: elastic
+password: changeme
+index: "some-index"
+pipeline: "some-ingest-pipeline"
+proxy_url: "https://proxy.url"
+backoff:
+  init: 42s
+  max: 420s
+workers: 30
+headers:
+  X-Header-1: foo
+  X-Bar-Header: bar`
+
+		basicOp := `
```

Comment:
This name here is also confusing me a bit. What does `op` stand for here, operation?

What about renaming it to `otelOutputCfg`?
```suggestion
		otelOutputCfg := `
```

Reply:
The `Op` stood for output. I'll rename it to keep it consistent. Thanks for pointing it out